lib/types/swap.nix

```
@@ -51,6 +56,7 @@
           device = config.device;
           randomEncryption = config.randomEncryption;
         }];
+        boot.resumeDevice = lib.mkIf config.resumeDevice dev;
```

I think this now failing because dev was moved:

https://github.com/nix-community/disko/pull/284/files#diff-11eba76ae08259c700ccca82f07b09532be9430b67a5b964f553f3d1265efd16R77

```suggestion
        boot.resumeDevice = lib.mkIf config.resumeDevice config.device;
```